openra/management/commands/import_latest_engines.py

```diff
@@ -0,0 +1,73 @@
+from dependency_injector.wiring import Provide, inject
+from django.core.management.base import BaseCommand, CommandError
+import github
+from openra.containers import Container
+import re
+from openra.services.engine_provider import EngineProvider
+
+from openra.services.github import Github
+
+class Command(BaseCommand):
+    help = 'Pulls the latest engines from GitHub'
+
+    def add_arguments(self, parser):
+        pass
+
+    def handle(self, *args, **options):
+        releases = self._get_latest_releases()
+
+        self._download_releases(releases)
+
+    @inject
+    def _get_latest_releases(self, github:Github=Provide[Container.github]):
+        playtest_regex = re.compile('^playtest-')
```

Comment:
This won't include/exclude devtests. (I don't quite understand what this method is trying to do though.)

Reply:
I think you are looking at an out-of-date version of this function, as it determines whether it's a playtest by looking at the "prerelease" flag on github now.

The idea of the function is to get the latest release, and if there are playtests more recent than it, get the latest playtest too.

In the latest version, you can specify a number of releases to get. It will get this number of full releases, and only get a playtest if it is the most recent "release".

Reply:
Ah. I'm going through this commit by commit.